src/Connections/GithubRepo.ts

```diff
@@ -660,10 +660,30 @@ export class GitHubRepoConnection extends CommandConnection<GitHubRepoConnection
             users = users[0].split(",");
         }
 
+        if (users.length === 0) {
+            // Assume self.
+            users = [(await octokit.users.getAuthenticated()).data.login];
+        }
+
+        let issueNumber = number && parseInt(number, 10);
+        if (!issueNumber) {
```

Comment:
Does it matter that passing 0 or a non-number value is treated the same as passing no number?

If it's worth anything, passing a negative number explicitly fails.